mteb/abstasks/clustering.py

```diff
@@ -53,6 +55,17 @@ def _evaluate_clustering_bootstrapped(
         - A dictionary where keys are level names (e.g., "Level 0", "Level 1", etc.) and values are lists of V-measure scores for each clustering experiment at that level.
         - A dictionary where keys are level names and values are lists of cluster assignments for each clustering experiment at that level.
     """
+    # set OMP_NUM_THREADS for reproductibility
```

Comment:
Can we add a sentence to describe _how_ this ensures reproducibility please?

Reply:
I will merge this for now, just to see if it resolves some of the flakiness - if it does not, I will roll it back.